### PR TITLE
DEV-284 Build logs

### PR DESCRIPTION
**Pull request by drekle**
Proposed message structure for retrieving site build logs.

### CODE REVIEW

File: live/sites/v1alpha1/site.proto [hunk 1]
```diff
@@ -701,6 +701,45 @@ message MysqlLogsResponse {
...
+  `OutputOnly` An array of scoped logs
+  */
+  repeated ScopedLogs logs = 1;
+}
+
+message ScopedLogs {
+  /*
+  `OutputOnly` A named scope to which these log messages apply
+  */
+  string scope = 1;
```
Comment (wozniakjan) [inline live/sites/v1alpha1/site.proto:735]: scope are the `steps` in tekton `taskrun` (the build pod containers)?

**Review by wozniakjan — APPROVED** [review]
lgtm

File: live/sites/v1alpha1/site.proto (hunk 1 above)
Comment (drekle) [reply live/sites/v1alpha1/site.proto:735]: That's the idea, however granting us flexibility in not using container name here as that may not be immediately understood by a user, and imagining that down the line we may have other log requests that have multiple scopes, such as mysqlserver databases or something that I cannot plan for at this moment.

**Comment by wozniakjan** [comment]
this looks like it needs just very little to move forward, I will rebase, test and try to get it merged

**Review by nic-hima — APPROVED** [review]
lgtm

**Pull request merged**